dist/htmlminifier.js

```diff
@@ -1698,7 +1698,7 @@
     return minify(value, options);
   };
 
-}(typeof exports === 'undefined' ? this : exports));
+})(typeof exports === 'undefined' ? this : exports);
```

Comment:
I'll have to edit some files either way `htmlparser.js` is `inline` - but `outside` it is then :wink: